nipoppy_cli/docs/source/conf.py

```diff
@@ -76,6 +76,18 @@
 autoapi_own_page_level = "class"
 autoapi_template_dir = "_templates/autoapi"
 
+# ignore some auto doc related warnings
+#  see https://github.com/sphinx-doc/sphinx/issues/10785
```

Comment:
from my reading of the issues I have found this seems like it will be a headache to fix so for now I am just silencing the warnings